historian-server/src/main/java/com/hurence/webapiservice/http/api/grafana/modele/HurenceDatasourcePluginQueryRequestParam.java

```diff
@@ -92,6 +94,7 @@ public String getRequestId() {
         private int bucketSize = DEFAULT_BUCKET_SIZE;
         private Map<String, String> tags = DEFAULT_TAGS;
         private String requestId = DEFAULT_REQUEST_ID;
+        private List<AGG> aggreg;
```

Comment:
Don't you see a difference with others properties ?

Reply:
yes the default